include/rospack/rospack.h

```diff
@@ -152,6 +152,7 @@ class ROSPACK_DECL Rosstackage
     std::tr1::unordered_map<std::string, Stackage*> stackages_;
     Stackage* findWithRecrawl(const std::string& name);
     void log(const std::string& level, const std::string& msg, bool append_errno);
+    void clearStackages();
```

Comment:
I also was afraid of breaking ABI. However, [here](https://techbase.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B#The_Do.27s_and_Don.27ts) it was stated that

  *adding new non-virtual functions*

is no problem for ABI compatibility. This finding was supported by [several other sources](https://www.google.de/search?q=abi+compatibility+adding+method&oq=abi+&aqs=chrome.0.69i59j0l2j69i57j0l2.1878j0j7&client=ubuntu&sourceid=chrome&es_sm=122&ie=UTF-8) too.